core-java-modules/core-java-date-operations-4/src/test/java/com/baeldung/timestamptocalendar/SqlTimestampToCalendarConverterUnitTest.java

```diff
@@ -0,0 +1,33 @@
+package com.baeldung.timestamptocalendar;
+
+import static org.junit.Assert.assertEquals;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.Calendar;
+
+import org.junit.Test;
+
+public class SqlTimestampToCalendarConverterUnitTest {
+
+    @Test
+    public void givenTimestamp_whenConvert_thenEqualMillis() {
+        Timestamp timestamp = new Timestamp(124, 3, 19, 9, 30, 0, 801789562);
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTimeInMillis(timestamp.getTime());
+        assertEquals(calendar.getTimeInMillis(), timestamp.getTime());
+        System.out.println(timestamp);// 2024-04-19 09:30:00.801789562
```

Comment:
As per the guidelines, if you need to print values, use an SLF4J logger. Alternatively, remove the logging and rely on assertions instead.